src/material-experimental/mdc-snack-bar/_snack-bar-theme.scss

```diff
@@ -31,6 +31,12 @@
   $mdc-snackbar-fill-color: $orig-mdc-snackbar-fill-color !global;
   $mdc-snackbar-label-ink-color: $orig-mdc-snackbar-label-ink-color !global;
   $mdc-snackbar-dismiss-ink-color: $orig-mdc-snackbar-dismiss-ink-color !global;
+
+  .mat-mdc-simple-snack-bar .mdc-snackbar__action {
```

Comment:
Could we achieve the same by setting `color="accent"` on the button in `simple-snack-bar.html` or will that not work correctly in dark themes?

Reply:
My thought was that it's possible, albeit really unlikely, that someone imports the snackbar theme but never imports the button theme.